### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by thwjanssen89**
Not a scam

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -7911,7 +7910,7 @@
 		"polkadots-drop.com",
 		"polkadots-event.com",
 		"polkadots.js.org",
-		"polkadots.live",
```
Comment (jacogr) [inline all.json:7914]: Was this one meant to be removed? There is nothing here at this point in time.

File: all.json [hunk 2]
```diff
@@ -7911,7 +7910,7 @@
...
+		"ps.live",
```
Comment (jacogr) [inline all.json:7913]: Parked domain at this point.

File: all.json (hunk 1 above)
Comment (thwjanssen89) [reply all.json:7914]: No only this one: polkadot-ui.authright-test.newtouch.com, I wouldn't know about the other ones

Comment (jacogr) [reply all.json:7914]: Yes, indeed, the polkadot-ui.authright-test.newtouch.com one needs removal (as done above), just asking about polkadots.live here specifically which has also been removed here.

Comment (thwjanssen89) [reply all.json:7914]: I thought I only deleted the polkadot-ui one. Also seems that way in the commit too me. I didn't mean to delete any other URL

Comment (jacogr) [reply all.json:7914]: The other one was removed/changed, see 

image

... hence my question :)

**Review by jacogr — APPROVED** [review]
Sorry for being a PITA.

**Pull request merged**